spec/models/post_spec.rb

```diff
@@ -1905,4 +1905,20 @@ def updates_topic_updated_at
       expect(post4.canonical_url).to eq("#{topic_url}?page=2#post_#{post4.post_number}")
     end
   end
+
+  describe ".show_welcome_topic_banner?" do
+    it "returns true when conditions are met" do
```

Comment:
One general tip I picked up from @nattsw recently is that we should try and be more specific in our test descriptions. In this case, conditions is very vague it is not clear what the actual conditions are.